webapp/src/styles/_utils.scss

```diff
@@ -93,3 +121,9 @@
 .overflow-auto {
     overflow: auto;
 }
+
+.text-truncate {
```

Comment:
`text-ellipses` class is used in a few places but that util class isn't there in the util styles.